wolfhsm/wh_common.h

```diff
@@ -35,7 +35,7 @@
 /** Resource allocations */
 enum WOLFHSM_NUM_ENUM {
     WOLFHSM_NUM_COUNTERS = 8,       /* Number of non-volatile 32-bit counters */
-    WOLFHSM_NUM_RAMKEYS = 16,        /* Number of RAM keys */
+    WOLFHSM_NUM_RAMKEYS = 8,        /* Number of RAM keys */
```

Comment:
Agree that these should be overridable at compile time by the user and likely placed in a seperate wh_config.h file.  Need to remove these from the enum and make then #defines at some point then.  Ok for now